I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 707, in script
    H "Siang murid-murid semua."
  File "game/script.rpy", line 253, in execute
    screen pilihanbuku:
  File "game/script.rpy", line 253, in execute
    screen pilihanbuku:
  File "game/script.rpy", line 254, in execute
    frame:
  File "game/script.rpy", line 276, in execute
    vbox:
  File "game/script.rpy", line 278, in execute
    imagebutton:
Exception: Imagebutton does not have a idle image.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 707, in script
    H "Siang murid-murid semua."
  File "I:\RenPy\renpy-8.0.3-sdk\renpy\ast.py", line 930, in execute
    renpy.exports.say(who, what, *args, **kwargs)
  File "I:\RenPy\renpy-8.0.3-sdk\renpy\exports.py", line 1473, in say
    who(what, *args, **kwargs)
  File "I:\RenPy\renpy-8.0.3-sdk\renpy\character.py", line 1349, in __call__
    self.do_display(who, what, cb_args=self.cb_args, dtt=dtt, **display_args)
  File "I:\RenPy\renpy-8.0.3-sdk\renpy\character.py", line 1000, in do_display
    display_say(who,
  File "I:\RenPy\renpy-8.0.3-sdk\renpy\character.py", line 690, in display_say
    rv = renpy.ui.interact(mouse='say', type=type, roll_forward=roll_forward)
  File "I:\RenPy\renpy-8.0.3-sdk\renpy\ui.py", line 299, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "I:\RenPy\renpy-8.0.3-sdk\renpy\display\core.py", line 3499, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, pause=pause, pause_start=pause_start, pause_modal=pause_modal, **kwargs) # type: ignore
  File "I:\RenPy\renpy-8.0.3-sdk\renpy\display\core.py", line 3969, in interact_core
    root_widget.visit_all(lambda d : d.per_interact())
  File "I:\RenPy\renpy-8.0.3-sdk\renpy\display\core.py", line 605, in visit_all
    d.visit_all(callback, seen)
  File "I:\RenPy\renpy-8.0.3-sdk\renpy\display\core.py", line 605, in visit_all
    d.visit_all(callback, seen)
  File "I:\RenPy\renpy-8.0.3-sdk\renpy\display\core.py", line 605, in visit_all
    d.visit_all(callback, seen)
  File "I:\RenPy\renpy-8.0.3-sdk\renpy\display\screen.py", line 476, in visit_all
    callback(self)
  File "I:\RenPy\renpy-8.0.3-sdk\renpy\display\core.py", line 3969, in <lambda>
    root_widget.visit_all(lambda d : d.per_interact())
  File "I:\RenPy\renpy-8.0.3-sdk\renpy\display\screen.py", line 487, in per_interact
    self.update()
  File "I:\RenPy\renpy-8.0.3-sdk\renpy\display\screen.py", line 680, in update
    self.screen.function(**self.scope)
  File "game/script.rpy", line 253, in execute
    screen pilihanbuku:
  File "game/script.rpy", line 253, in execute
    screen pilihanbuku:
  File "game/script.rpy", line 254, in execute
    frame:
  File "game/script.rpy", line 276, in execute
    vbox:
  File "game/script.rpy", line 278, in execute
    imagebutton:
  File "I:\RenPy\renpy-8.0.3-sdk\renpy\ui.py", line 948, in _imagebutton
    idle = choice(idle, idle_image, "idle", required=True)
  File "I:\RenPy\renpy-8.0.3-sdk\renpy\ui.py", line 944, in choice
    raise Exception("Imagebutton does not have a %s image." % (name,))
Exception: Imagebutton does not have a idle image.

Windows-10-10.0.19045 AMD64
Ren'Py 8.1.0.23051307
 
Sun Jun  4 08:56:42 2023
